en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -231,8 +231,9 @@ See [silverstripe/vendor-plugin](https://github.com/silverstripe/vendor-plugin)
 
 Silverstripe CMS relies on various [caches](/developer_guides/performance/caching/)
 to achieve performant responses. By default, those caches are stored in a temporary filesystem folder, and are not
-shared between multiple server instances. Alternative cache backends such as Redis can be
-[configured](/developer_guides/performance/caching/).
+shared between multiple server instances.
+
+No in-memory cache is used by default. We strongly recommend [configuring an in-memory cache](/developer_guides/performance/caching/#adapters) such as Redis or Memcached.
```

Comment:
```suggestion
No in-memory cache is used by default. To improve performance on larger applications we recommend [configuring an in-memory cache](/developer_guides/performance/caching/#adapters) such as Redis or Memcached.
```
It's a performance optimisation, it's not the end of the world if this isn't done.

Reply:
It's not the end of the world, but it should still be a strong recommendation. People can choose not to do it, but generally they should do it.

It's also not just for larger applications - it will always be an improvement for any size project.

Reply:
The problem here is that this by saying this something your really should do, it's essentially adding an extra barrier to adoption of Silverstripe for new projects. We want encourage people to get up and running as easily as possible, not burden them with extra things they need to do.

A lot of people also have very little control over their server environment. By making redis/memcached a "strong recommendation" then we're kind of saying that Silverstripe is perhaps not an appropriate choice for them, which simple is not the case. Silverstripe works just fine without either of those.

Reply:
Fair enough, will make the change - but I'll leave out the "on larger applications" part.

Reply:
Done